unit-test/unit-test.md

```diff
@@ -308,3 +268,55 @@ K cell will be empty when the main semantics abort. However, we know that the
   rule <k> Aborted(_, _, Rk, _) #as V => #FindVerifyOutput(Rk, V) ... </k>
 endmodule
 ```
+
+This function implements a relaxed equality check between two data elements. In
+particular, it handles the wildcard matching behavior described in the .tzt
+format proposal and discards list type information as discussed earlier.
+
+```k
+module MATCHER
+  imports MICHELSON-COMMON
+  imports UNIT-TEST-SYNTAX
+
+  syntax Bool ::= #Matches(Data, Data) [function] // Expected, Actual
+
+  rule #Matches(#Any, _) => true
+
+  rule #Matches(D1, D2) => D1 ==K D2 [owise]
+  // This also covers any structurally different data. (e.g. (Left 1) vs (Right 1))
```

Comment:
What does this comment mean? Perhaps the RHS is intended to be `#Matches(D1, D2)`, but I guess thats not a change made by this PR

Reply:
Any case that isn't explicitly covered reduces to built-in equality.